### DISCUSSION TITLE
Storybook <Stories /> doc block in mdx no longer rendering after upgrading to version 8.1.0

### DISCUSSION THREAD

**Discussion by ryanelian** [storybookjs/storybook#27142] [Help]
### Summary

I have stories written like this:

> RouterProvider.mdx

```mdx
import { Canvas, Meta } from '@storybook/blocks';

import { Title, Subtitle, Description, Primary, Controls, Stories, Source } from '@storybook/blocks';

import RouterProviderStories from './RouterProvider.stories';

<Meta of={RouterProviderStories} />

<Title />

<Subtitle />

Redacted

<Stories />
```

> RouterProvider.stories.tsx

```tsx
import type { Meta } from '@storybook/react';

import { RouterProvider } from 'react-aria-components';

// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
const meta = {
  title: 'Components/RouterProvider',
  component: RouterProvider,
  parameters: {
    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/configure/story-layout
    layout: 'centered',
  },
} satisfies Meta<typeof RouterProvider>;

export default meta;

export { RouterProviderStory } from './RouterProvider.story';
```

Where that story re-exported from another file is a way for us to customize the rendering of the component and display source code appropriate for our use case:

```tsx
import component from './RouterProvider.story.source';
import sourceCode from './RouterProvider.story.source?raw';

export const RouterProviderStory = {
  name: 'RouterProvider',
  render: component,
  parameters: {
    layout: 'centered',
    storySource: {
      source: sourceCode,
    },
    design: [
      {
        type: 'link',
        url: 'https://react-spectrum.adobe.com/react-aria/routing.html',
      },
    ],
    docs: {
      source: {
        code: sourceCode,
      },
    },
  },
};
```

The component being rendered and the source code of the component is not relevant for this discussion, and therefore not included to not distract the focus from the core issue.

When updating to version 8.1.0, these MDX `<Stories />` block failed to render.

### Additional information

For some reason, stories generated by autodocs `tags: ['autodocs'],` gets story blocks generated correctly.

But stories that are hand-written using MDX do not.

**Comment by shilman** [comment]
Aha, sorry about that. It's an unintentional breaking change due to https://github.com/storybookjs/storybook/pull/26634

I'll chat with the team and figure out a workaround. Following up in https://github.com/storybookjs/storybook/issues/27143
